crates/typos-cli/src/config.rs

```diff
@@ -98,6 +98,8 @@ impl Config {
 #[serde(rename_all = "kebab-case")]
 pub struct Walk {
     pub extend_exclude: Vec<String>,
+    // Exclude files even if passed explicitly.
+    pub force_exclude: Option<bool>,
```

Comment:
To me, this feature seems more situational rather than being a fixed state, so I think it should be cli only and not in config